package/system/apk/Makefile

```diff
@@ -0,0 +1,56 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=apk
+PKG_VERSION:=3e6261392f990b541effe94b2515456e977bc79b
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=apk-tools-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://gitlab.alpinelinux.org/alpine/apk-tools/-/archive/$(PKG_VERSION)
+PKG_HASH:=5fd2c8b9272f803760c377ac0f95ca3eec0ad6f6a29a3428e8319ac245ad3348
+PKG_BUILD_DIR:=$(BUILD_DIR)/apk-tools-$(PKG_VERSION)
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/apk-tools-$(PKG_VERSION)
+
```

Comment:
You still need `HOST_BUILD_DEPENDS := lua/host lua-lzlib/host` here to build the help database.

Reply:
Added, the missing bit was `lua_version=5.1`